tests/test_spawn.rs

```diff
@@ -183,6 +183,7 @@ pub fn test_spawn_int() {
     assert!(result.is_ok());
     assert!(result.unwrap() == 0);
     // When the VM makes 64 recursive calls, make sure the stack is less than 1M.
+    println!("stack size: {}k", (cur_sp - *min_sp.lock().unwrap()) / 1024);
```

Comment:
Personally I would recommend printing in raw bytes, we could complement this value with a percentage of the 1MB budgets. My thought is that some change might only affect a few bytes, always printing this value in KB, might hinder some changes.